.github/workflows/api-deployer.yml

```
@@ -173,18 +173,19 @@ jobs:
           echo "POSTGRES_HOST=localhost" >> config/.env.local
           echo "ENV=dev" >> config/.env.local
 
-      - name: Docker Compose DB/Liquibase for db-gen.sh
-        run: docker-compose --env-file ./config/.env.local up -d liquibase
-        working-directory: ${{ github.workspace }}
+      - uses: actions/download-artifact@v4
+        with:
+          name: database_gen
+          path: api/src/database_gen/
 
-      - name: Generate DB code
+      - name: Copy to db models to functions directory
```

Maybe add a comment. I had to hunt to know where are the models generated
```suggestion
      # db models were generated aand uploaded in api-build-test job above.
      - name: Copy to db models to functions directory
```